src/main/java/com/algorand/sdkutils/listeners/JavaGenerator.java

```diff
@@ -378,20 +379,28 @@ public void storeEnumDefinition(TypeDef typeObj) {
             String javaEnum = Tools.getCamelCase(val, true).toUpperCase();
             sb.append(javaEnum);
             sb.append("(\"" + val + "\")");
-            if (elmts.hasNext()) {
-                sb.append(",\n");
-            } else {
-                sb.append(";\n\n");
-            }
+            sb.append(",\n");
         }
+        sb.append(TAB + TAB + "@JsonProperty(\"\") UNKNOWN(\"\");\n\n");
```

Comment:
@winder For my understanding, why is `@JsonProperty` required?

Reply:
In this PR I mainly followed the existing pattern and didn't consider it further once I had a test passing. I can give a rough explanation though.

This an annotation is part of the [Jackson](https://github.com/FasterXML/jackson-docs) library, which is widely used in the industry for serializing/deserializing objects. We're also using an extension that supports msgpack.

It's probably optional for the enum types which are using the same name as the enum type.

Reply:
@winder Thanks for the response - there's enough here for me to resolve.

Yes - the last statement is what I'm gauging.  Since the configured annotation value provides _no_ runtime change, it's unclear to me that it's _needed_.

I'm OK to leave it in place because I'm unsure (particularly without tests) what other side effects removing the annotation may cause.